### PR TITLE
Pika/style/hide and seek background

### PR DESCRIPTION
**Pull request by priskilladrn**

### CODE REVIEW

File: Macrow-ADHD/Features/Games/HideAndSeek/HideAndSeekBackground.swift [hunk 1]
```diff
@@ -30,64 +30,65 @@ class BackgroundHideAndSeek: SKNode {
     }
     
     func addBackground() {
-        background = .init(imageNamed: "Background 1")
-        background.position = CGPoint(x: sceneFrame.width / 2 - 80, y: sceneFrame.height * 0.11)
-        background.setScale(1)
-        background.zPosition = 5
+        land1 = .init(imageNamed: "Land 1")
```
Comment (gregoriusyuristama) [inline Macrow-ADHD/Features/Games/HideAndSeek/HideAndSeekBackground.swift]: pake resourcepath jangan diketik pake petik (")

**Review by gregoriusyuristama — APPROVED** [review]
good

**Review by gregoriusyuristama — CHANGES_REQUESTED** [review]
perlu ganti pake resource path

**Review by azellaa — APPROVED** [review]
good

**Pull request merged**